File: functions/TestJs/handler.js
"use strict";

const aws_sdk = require("aws-sdk");
const uuid = require("uuid");
const elasticsearch = require("elasticsearch");
const axios = require("axios");

module.exports.main_function = async (event) => {
	return {
		statusCode: 200,
		body: JSON.stringify(
			{
				message: "Go Serverless v3.0! Your function executed successfully!",
				input: event,
        env: process.env
			},
			null,
			2
		),
	};
};

File: services/test-api-service/functions/TestPython/handler.py
import boto3
import json
import requests
from requests_aws4auth import AWS4Auth
from pprint import pprint
import os

is_offline = os.getenv("IS_OFFLINE", "false")


response = {
    "statusCode": 400,
    "headers": {
        'Content-Type': 'application/json',
        "Access-Control-Allow-Headers": "*",
        "Access-Control-Allow-Origin": "*",
        "Access-Control-Allow-Methods": "*",
    },
}


def main_function(event, context):
    try:

        response['statusCode'] = 200
        response["body"] = json.dumps({
            "message": "Go Serverless v3.0! Your function executed successfully!",
            "env": dict(os.environ),
            "event": event,
        })

    except Exception as e:
        print(e)
        response["body"] = json.dumps({"message": str(e)})

    return response
